### PR TITLE
Fixes for issue "Fix the LGTM issues #1024"

### PR DESCRIPTION
**Pull request by brunohadlich**

### CODE REVIEW

File: project_euler/problem_03/sol1.py [hunk 1]
```diff
@@ -28,18 +28,22 @@ def isprime(no):
...
-    
+
     >>> solution(13195)
     29
     >>> solution(10)
     5
     >>> solution(17)
     17
+    >>> solution(0)
+    -1
```
Comment (cclauss) [inline project_euler/problem_03/sol1.py]: Please consider adding a negative integer, a floating point number, an empty list, and a string.

**Comment by cclauss** [comment]
Related to #1024

File: project_euler/problem_03/sol1.py (hunk 1 above)
Comment (brunohadlich) [reply project_euler/problem_03/sol1.py]: My last commit adds code to handle such situations as well as doctests.

**Comment by cclauss** [comment]
Test failure

**Pull request merged**